configs/components/facter.rb

```diff
@@ -110,7 +110,7 @@
     skip_blkid = 'OFF'
   elsif platform.is_rpm?
     if (platform.is_el? && platform.os_version.to_i >= 6) || (platform.is_sles? && platform.os_version.to_i >= 11) || platform.is_fedora?
-      pkg.build_requires "libblkid-devel" unless platform.architecture == "s390x"
+      pkg.build_requires "libblkid-devel" unless platform.is_cross_compiled?
```

Comment:
This won't catch any additional things, will it? Are we not cross-compiling anything else besides s390x and now this new one?